packages/util-node-http-modules/src/util-node-http-modules.test.js

```diff
@@ -1,6 +1,3 @@
-import {getNodeHttpModules} from "./util-node-http-modules"
-
-test("HTTP Transports exist", async () => {
-  const {nodeHttpsTransport, nodeHttpTransport} = await getNodeHttpModules()
-  expect(nodeHttpTransport && nodeHttpsTransport).toBeDefined()
```

Comment:
Removed unit test because it was actually not doing anything before. Since our jest configuration does not support experimental ES modules, nodeHttpTransport && nodeHttpsTransport was null anyways. The .toBeDefined check passes with null and really should have been a .toBeTruthy check.